index.js

```diff
@@ -25,6 +25,18 @@ let currentWordIndex = 0;
 
 // todo: add shuffle button
 // todo: submit function
+
+const submitBtn = document.getElementById("validateCurrentWord");
+submitBtn.addEventListener("click", () => {
+  const wordEnterByUser = document.getElementById("wordToCheck").value.toLowerCase();
+  const currentWord = wordsToPlay[currentWordIndex].toLowerCase();
```

Comment:
Please remove `toLowerCase()` on both words because this practice is case sensitive.